src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -1158,8 +1158,11 @@ public void run()
         }
         catch (SocketException ex)
         {
-            if (!"Socket closed".equals(ex.getMessage()))
+            if (!"Socket closed".equals(ex.getMessage())
```

Comment:
i'm just wondering why we have to do the string comparison and why the 'instanceof' check isn't enough?